File: Ammamentar.Domain/Pessoa.cs
using System;
using System.Collections.Generic;

namespace Ammamentar.Domain
{
    public class Pessoa
    {
        public int Id { get; set; }
        public string Nome { get; set; }
        public string Email { get; set; }
        public string Senha { get; set; }
        public string Cpf { get; set; }
        public string Rg { get; set; }
        public string Telefone { get; set; }
        public string Celular { get; set; }
        public int RegistroAtivo { get; set; }
        public DateTime DataCadastro { get; set; }
        public List<Endereco> Enderecos { get; set; }
    }
}

File: Ammamentar-App/src/app/_models/Pessoa.ts
export interface Pessoa {

    id: number;
    nome: string;
    email: string;
    senha: string;
    cpf: string;
    rg: string;
    telefone: string;
    celular: string;
    registroAtivo: number;
    dataCadastro: Date;
    enderecos: Endereco[];
}
